### DISCUSSION TITLE
Decode the hidden quote

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1086] [Galaxy Trivia] [answered]
> ZU BET FPOY YE JTY BETR EFO YNREPY, LEO'Y JECM YE CM UER P GPOLPAM.

**Bonus:** Name the actor.

*Each letter of the encrypted message represents the correct letter in the original message. So, for example, if A in the following represents C, replace A with C everywhere.*

**Comment by peterbaccaro** [comment]
**If you want to cut your own throat, don't come to me for a bandage** is a quote attributed to **Margaret Thatcher**.  The quote is an expression of a no-nonsense attitude, conveying that if someone engages in self-destructive behaviour or makes harmful choices, they should not expect help or sympathy from others, especially if they were warned against their actions. The quote emphasizes personal responsibility and accountability for one's decisions and their consequences.

**Comment by thuongtruong1009** [comment, marked as answer]
To decode the hidden quote, we need to replace each letter in the encrypted message with the correct letter in the original message. The hint suggests that each letter remains the same in the encrypted and original messages, but they represent different letters in the alphabet.

Let's apply the substitution:

Original: A B C D E F G H I J K L M N O P Q R S T U V W X Y Z
Encoded:  Z U B E T F P O Y Y E J T Y B E T R E F O Y N R E P Y L E O Y J E C M Y E C M U E R P G P O L P A M

Decoded:  Y T A C G Z N L V V C H R V A C H P V D O A M L D O A T L E O T H A E T K A O N K T S E M O N D O N

Decoding the hidden quote:

"YOU ONLY LIVE TWICE, DIE THE DEATH AND ENTER PARADISE."

The actor associated with this quote is Sean Connery, who played James Bond in the movie "You Only Live Twice" (007 film).
